dotnet/src/Connectors/Connectors.UnitTests/Memory/Pinecone/PineconeMemoryStoreTests.cs

```diff
@@ -223,7 +223,7 @@ public async Task TestGetNearestMatchesAsync()
         // Arrange
         Embedding<float> embedding = new(new float[] { 0.1f, 0.2f });
 
-        List<(PineconeDocument, double)> queryResults = new List<(PineconeDocument, double)>
+        List<(PineconeDocument, double)> queryResults = new()
```

Comment:
this is IDE0090: if that's too futuristic I can undo. Personally I like it, it's both explicit and concise

Reply:
This is one of my favs.